### PR TITLE
Use map instead of from to create a new Uint8Array

### PR DESCRIPTION
**Pull request by teoli2003**
See https://github.com/mdn/content/pull/19823#issuecomment-1222394840

Note that I can confirm that:
- it is simpler, much easier to understand
- much quicker on V8 (I have a 4x difference!)
- as quick as `from` on Gecko (< 1% difference), which is 50% quicker than V8.
- 
Thanks, @zloirock

### CODE REVIEW

File: files/en-us/web/api/xmlhttprequest/sending_and_receiving_binary_data/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ tags:
   - MIME
   - XMLHttpRequest
 ---
-The `responseType` property of the XMLHttpRequest object can be set to change the expected response type from the server. Possible values are the empty string (default), `"arraybuffer"`, `"blob"`, `"document"`, `"json"`, and `"text"`. The `response` property will contain the entity body according to `responseType`, as an `ArrayBuffer`, `Blob`, `Document`, `JSON`, or string. This is `null` if the request is not complete or was not successful.
+The `responseType` property of the XMLHttpRequest object can be set to change the expected response type from the server. Possible values are the empty string (default), `"arraybuffer"`, `"blob"`, `"document"`, `"jsonEt "`, and `"text"`. The `response` property will contain the entity body according to `responseType`, as an `ArrayBuffer`, `Blob`, `Document`, `JSON`, or string. This is `null` if the request is not complete or was not successful.
```
Comment (Josh-Cena) [inline files/en-us/web/api/xmlhttprequest/sending_and_receiving_binary_data/index.md:11]: This?

File: files/en-us/web/api/xmlhttprequest/sending_and_receiving_binary_data/index.md [hunk 2]
```diff
@@ -96,13 +96,12 @@ req.send(blob);
 You can send JavaScript typed arrays as binary data as well.
 
 ```js
-const array = new ArrayBuffer(512);
 // Create a new array with fake data (Consecutive numbers (0 - 255), looping back to 0) 
-const longInt8View = Uint8Array.from(array, (v, i) => i % 256);
+const largeUInt8Array = new Uint8Array(512).map((v, i) => i % 256);
```
Comment (zloirock) [inline files/en-us/web/api/xmlhttprequest/sending_and_receiving_binary_data/index.md:100]: `% 256` is not required.

File: files/en-us/web/api/xmlhttprequest/sending_and_receiving_binary_data/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/xmlhttprequest/sending_and_receiving_binary_data/index.md:11]: I wondered where this letters went.
```suggestion
The `responseType` property of the XMLHttpRequest object can be set to change the expected response type from the server. Possible values are the empty string (default), `"arraybuffer"`, `"blob"`, `"document"`, `"json"`, and `"text"`. The `response` property will contain the entity body according to `responseType`, as an `ArrayBuffer`, `Blob`, `Document`, `JSON`, or string. This is `null` if the request is not complete or was not successful.
```

File: files/en-us/web/api/xmlhttprequest/sending_and_receiving_binary_data/index.md (hunk 2 above)
Comment (teoli2003) [reply files/en-us/web/api/xmlhttprequest/sending_and_receiving_binary_data/index.md:100]:
```suggestion
const largeUInt8Array = new Uint8Array(512).map((v, i) => i);
```

File: files/en-us/web/api/xmlhttprequest/sending_and_receiving_binary_data/index.md [hunk 3]
```diff
@@ -96,13 +96,12 @@ req.send(blob);
 You can send JavaScript typed arrays as binary data as well.
 
 ```js
-const array = new ArrayBuffer(512);
 // Create a new array with fake data (Consecutive numbers (0 - 255), looping back to 0) 
-const longInt8View = Uint8Array.from(array, (v, i) => i % 256);
+const largeUInt8Array = new Uint8Array(512).map((v, i) => i);
```
Comment (Josh-Cena) [inline files/en-us/web/api/xmlhttprequest/sending_and_receiving_binary_data/index.md:100]: It's interesting to see how this example drifted. This `longInt8View` was initially intended to be a _view_ of that `ArrayBuffer` but when we refactored it to `Uint8Array.from()`, that semantic was lost.

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**